File: src/Operator.java
/**
 * Клас Оператор станції
 */
public class Operator extends Person {
    private String stationId;

    public Operator(String name, String stationId) {
        super(name, "Оператор станції");
        this.stationId = stationId;
    }

    public void monitor() {
        System.out.println("Оператор " + getName() + " проводить моніторинг станції " + stationId);
    }

    public void report() {
        System.out.println("Оператор " + getName() + " формує звіт по станції " + stationId);
    }
}

File: src/Operator.py
# Клас Оператор станції
class Operator(Person):
    def __init__(self, name: str, station_id: str):
        super().__init__(name, "Оператор станції")
        self.station_id = station_id

    def monitor(self):
        print(f"Оператор {self.get_name()} проводить моніторинг станції {self.station_id}")

    def report(self):
        print(f"Оператор {self.get_name()} формує звіт по станції {self.station_id}")
